.github/workflows/release.yml

```diff
@@ -0,0 +1,434 @@
+      - run: git log --format="%H" -n 1
+      #- name: Extract version from tag
+      #  uses: damienaicheh/extract-version-from-tag-action@v1.3.0
+      - run: git tag
+      # Ok this needs to get up leveled somewhere so that it can be shared across Debian and Ubuntu obviously
+      - run: ls -lah
+      - name: Link the Ubuntu build stuff in as 'debian/'
+        run: ln -s debian.ubuntu debian
+      - name: Generate Ubuntu Changelog
+        run: |
+          echo "Major: ${MAJOR}"
+          echo "Minor: ${MINOR}"
+          echo "Patch: ${PATCH}"
+          echo "Num Patch: ${NUMBER_OF_COMMITS_SINCE_TAG}"
+          gbp dch \
+            --debian-branch ${{ needs.tempbranch.outputs.rc_ref }} \
+            -R \
+            -N ${MAJOR}.${MINOR}.${PATCH} \
+            --spawn-editor=never
+        env:
+          EMAIL: releases@tenstorrent.com
+          NAME: Tenstorrent Releases
+      - name: Link the Debian build stuff in as 'debian/'
+        run: rm debian
+      - run:  ln -s debian.debian debian
+      - run: ls -lah
+      - name: Generate Debian Changelog
+        run: |
+          gbp dch \
+            --debian-branch ${{ needs.tempbranch.outputs.rc_ref }} \
+            -R \
+            N ${MAJOR}.${MINOR}.${PATCH} \
+            --spawn-editor=never
+        env:
+          EMAIL: releases@tenstorrent.com
+          NAME: Tenstorrent Releases
+      - name: ok what happened
+        run: git diff
+      - name: Setup commit
+        run: |
+          git config --global user.email "${EMAIL}"
+          git config --global user.name "${NAME}"
+          git add .
+          git commit \
+            -m "Generating Ubuntu & Debian Changelog for ${MAJOR}.${MINOR}.${PATCH}-${NUMBER_OF_COMMITS_SINCE_TAG}"
+          git pull --rebase
+          git push
+        env:
+          EMAIL: releases@tenstorrent.com
+          NAME: Tenstorrent Releases
+      - name: Find New Git Hash
+        id: new-git-hash
+        run: |
+          export git_hash_env="git_hash=$( git log --format="%H" -n 1 )"
+          echo "${git_hash_env}"
+          echo "${git_hash_env}" >> "$GITHUB_OUTPUT"
+
+  ###
+  # Trigger Building Everything
+  ###
+  buildall:
+    needs:
+      - tempbranch
+      - versionchange
+      - changelogs
+    uses: ./.github/workflows/build-all.yml
+    with:
+      ref: ${{ needs.tempbranch.outputs.rc_ref }}
+      MAJOR: ${{ needs.versionchange.outputs.version_major }}
+      MINOR: ${{ needs.versionchange.outputs.version_minor }}
+      PATCH: ${{ needs.versionchange.outputs.version_patch }}
+      NUMBER_OF_COMMITS_SINCE_TAG: ${{ needs.versionchange.outputs.number_of_commits_since_tag }}
+    secrets: inherit
+
+  # Tag the Release
+  ###
+  tagrelease:
+    needs:
+      - versionchange
+      - changelogs
+      - buildall
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+    steps:
+      - name: dump the envs, I'm curious
```

Comment:
This is no longer necessary and probably a security risk.

Reply:
security risk: only if we do things very poorly, but yeah I'll drop this one